portal/server/routes/login.js

```diff
@@ -87,10 +87,12 @@ router.post('/reset-password', async (req, res) => {
   }
 });
 
+// User forgot password
 router.get('/reset-password/:pwdResetToken', (req, res) => {
-  res.render('user/change-password.njk', { requireCurrentPassword: false });
+  res.render('user/change-password.njk', { requireCurrentPassword: true });
```

Comment:
this is the route where they get a token in their email. at this stage they wouldn't have their current password.